app/models/user.rb

```diff
@@ -7,4 +7,16 @@ class User < ApplicationRecord
   PRE_RELEASE_FEATURES_PERMISSION = "pre_release_features"
   DEBUG_PERMISSION = "debug"
   MANAGING_EDITOR_PERMISSION = "managing_editor"
+  GDS_ACCESS_LIMIT_OVERRIDE_PERMISSION = "gds_access_limit_override"
```

Comment:
I'd suggest dropping the GDS prefix of this, there's nothing that forces it to be GDS and we have other permissions (debug, pre release) that don't have a prefix.

Reply:
Fair enough. My hope was to make it clear to the person applying it that it's only supposed to be used by GDS people, to help avoid it being applied accidentally. But it also seems reasonable that the people applying the permissions (who are currently entirely within GDS) should be aware of and responsible for their effects.

Reply:
Yup, I also think prefixing them with GDS slips it more towards role than effects. Thanks for changing.